internal/pipeline/testdata/simplepackage/.expected/diff.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/configs/resources.yaml b/configs/resources.yaml
+index 0dabe44..0124a57 100644
+--- a/configs/resources.yaml
++++ b/configs/resources.yaml
+@@ -2,6 +2,9 @@ apiVersion: apps/v1
+ kind: Deployment
+ metadata:
+   name: nginx-deployment
++  namespace: staging
++  labels:
++    tier: backend
+ spec:
+   replicas: 3
+ ---
+@@ -9,4 +12,8 @@ apiVersion: custom.io/v1
+ kind: Custom
+ spec:
+   image: nginx:1.2.3
++metadata:
```

Comment:
In the test framework for pipeline, do you run the formatter after running pipeline? This order of fields looks weird.

Reply:
updated it to run the formatter and changed the order of fields in the diff